readme.md

```diff
@@ -36,7 +36,19 @@ Of course, you can also find the CSFML API documentation in the SDK.
 
 ## Building
 
-Set `SFML_ROOT` to a directory containing the installation of SFML.
+Building CSFML follows typical CMake conventions.
+Here's how to get started.
+
+1. Install SFML.
+This may be done in a variety of ways including using a system package manager or building and installing SFML from source.
+The major version of SFML must match the major version of CSFML.
+1. Configure CSFML.
```

Comment:
Since Markdown files can/should also be readable in plaintext, I'd personally stick with continued numbering (1., 2., 3., ...)